uber/site_sections/groups.py

```diff
@@ -39,12 +39,13 @@ def _is_dealer_convertible(attendee):
     #     and c.DEALER_RIBBON in attendee.ribbon_ints
 
 
-def _decline_and_convert_dealer_group(session, group, delete_when_able=False):
+def _decline_and_convert_dealer_group(session, group, status=c.DECLINED):
```

Comment:
We now stop deleting dealers when they're declined, and instead set them to a "Declined" status. We can also set them to a Cancelled status, for cases when the dealer backs out. This should be basically invisible to admins, as we filter out declined groups in the groups view. I think.

Reply:
Just to clarify, we do filter out declined dealers when you're on the "With Tables" tab in the groups page.